pkg/sloop/webserver/webfiles.go

```diff
@@ -0,0 +1,51 @@
+package webserver
+
+import (
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"path"
+	"strings"
+	"sort"
+)
+
+const (
+	prefix = "webfiles/"
+)
+// go-bindata -o bindata.go webfiles
+// ReadWebfile is a function which finds the webfiles that have been predefined and converted to binary format.
+// sample input : filepath= "webfiles/index.html"
+func ReadWebfile(filepath string) ([]byte, error) {
```

Comment:
I think the precedence here should be external file first, then baked in form

Reply:
agreed